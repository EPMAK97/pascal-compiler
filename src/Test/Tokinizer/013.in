    1234567890
    f