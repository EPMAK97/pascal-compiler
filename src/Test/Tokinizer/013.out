1  |  5 | INTEGER     | CONST_INTEGER               | 1234567890| 1234567890
2  |  5 | IDENTIFIER  | VARIABLE                    | f         | 
END OF FILE
